test/spec/modules/appnexusAstBidAdapter_spec.js

```diff
@@ -129,6 +129,24 @@ describe('AppNexusAdapter', () => {
       delete REQUEST.bids[0].params.user;
     });
 
+    it('should add source and verison to the tag', () => {
+      REQUEST.sdk = {
+        source: 'pbjs',
+        version: '$prebid.version$'
+      };
```

Comment:
Setting this isn't required for the test to pass because the `sdk` object is added within the adapter. Calling `callBids` and examining the request as you have below should be enough

Reply:
Done